packages/file-menu/src/DeleteDialog.js

```diff
@@ -64,7 +64,14 @@ DeleteDialog.defaultProps = {
 
 DeleteDialog.propTypes = {
     open: PropTypes.bool,
-    fileType: PropTypes.oneOf(['chart', 'eventChart', 'reportTable', 'eventReport', 'map']),
+    fileType: PropTypes.oneOf([
```

Comment:
can we save all of these as a global const somewhere to avoid repetition?

Reply:
I added the list in `util.js`.